src/components/layout/Layout.tsx

```diff
@@ -26,28 +48,75 @@ const languages = [
   }
 ]
 
-const Okp4Link = ({ label }: FooterLinkProps): JSX.Element => {
+const Okp4Link = ({ label, url }: DeepReadonly<FooterLinkProps>): JSX.Element => {
   return (
```

Comment:
This return is not mandatory. You can also remplace the magic quote with {" "} for creating a space (up to you)

```suggestion
const Okp4Link = ({ label, url }: DeepReadonly<FooterLinkProps>): JSX.Element => (
    <Typography as="p" color="invariant-text" fontSize="x-small" fontWeight="xlight" noWrap>
      {label}{' '}
      <Typography color="invariant-text" fontSize="x-small" fontWeight="bold">
        <a className="okp4-brand-link" href={url} rel="author noreferrer" target="_blank">
          ØKP4
        </a>
      </Typography>
    </Typography>
  )
```

Reply:
For spaces I prefer to use template literal expressions than to have chaining expressions. I think it's more readable this way.

Good catch for the redundant return 👍